cljfmt/src/cljfmt/core.cljc

```diff
@@ -248,8 +248,30 @@
 (defn- reader-conditional? [zloc]
   (and (reader-macro? zloc) (#{"?" "?@"} (-> zloc z/down token-value str))))
 
+(defn- reader-conditional-form-symbol
+  "Given a zipper pointing to a reader conditional form like
+
+    #?(:clj potemkin.core/defprotocol+ :cljs defprotocol)
+
+  Return the first symbol inside it e.g.
+
+    potemkin.core/defprotocol+"
+  [zloc]
+  (when (reader-conditional? zloc)
+    ;; find the first keyword e.g. `:clj`
+    (when-let [key-loc (z/find (-> zloc z/down z/right z/down)
+                               z/right
+                               #(n/keyword-node? (z/node %)))]
+      ;; now move to the next non-whitespace/non-comment/non-metadata node after
+      ;; the keyword node e.g. `potemkin.core/defprotocol+`.
```

Comment:
I don't think we need this comment, the code seems clear enough, and if it isn't, we should split it out into more functions. Ideally comments should explain "why" rather than "what".

Reply:
done